docker-compose.yml

```
@@ -59,38 +58,21 @@ services:
       - ./relayminer:/home/pocket/.poktroll/
       - ./scripts/start-relayminer.sh:/home/pocket/start-relayminer.sh
 
-  # AppGate Server
-  appgate:
-    image: ${POKTROLLD_IMAGE_REPO}:${POKTROLLD_IMAGE_TAG_OFF_CHAIN:-latest}
-    container_name: appgate
-    entrypoint: ["sh", "/home/pocket/start-appgate.sh"]
-    environment:
-      - APPLICATION_MNEMONIC=${APPLICATION_MNEMONIC}
-    ports:
-      - "85:85" # 85 is the port AppGate Server is configured to listen on in `appgate_server_config.yaml`.
-    restart: unless-stopped
-    depends_on:
-      - full-node
-    volumes:
-      - /etc/ssl/:/etc/ssl/
-      - ./appgate:/home/pocket/.poktroll/
-      - ./scripts/start-appgate.sh:/home/pocket/start-appgate.sh
-
-  # Gateway
+  # PATH Gateway
+  # TODO_TECHDEBT: Rename the container to be consistent with both LocalNet and
```

If we don't give names, it'll never get done.

If it's actually something we care to have done, give it a DRI (DIrectly Responsible Individual)

```suggestion
  # TODO_TECHDEBT(@okdas): Rename the container to be consistent with both LocalNet and
```